include/wabt/type.h

```diff
@@ -46,6 +46,8 @@ class Type {
     FuncRef = -0x10,    // 0x70
```

Comment:
FuncRef is the same as `(ref null func)`, but `(ref func)` is not a possible value at the moment. I would use the index argument to store a bool like argument. I don't plan to do it now, but future tests needs it, so we can discuss this early.